index.js

```diff
@@ -188,6 +206,7 @@ const image = {
 
     // We support the file type
     token.consume(sentToken, matches[1], function (err) {
+        // OK #noKidding: Node-sqlite3 bind the `this`, so ES6 syntax is not possible here
         if (err || this.changes !== 1) {
```

Comment:
@TiddoLangerak I'm fine with that, we use it at only one place and I do not expect we will need that somewhere in the future :tm:

I love JS but this magic makes it sometimes less fun than drowning :|